docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -338,7 +319,66 @@ You can also look at the backup files:
    find /tmp/backups
 
 
-If `ansible-playbook` fails, please follow the debug steps in :doc:`network_debug_troubleshooting`.
+If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
+
+
+.. _network-agnostic-examples:
+
+Example 2: simplifying playbooks with network agnostic modules
+==============================================================
+
+If you have two or more network platforms in your environment, you can use the network agnostic modules to simplify your playbooks. You can use network agnostic modules such as ``cli_command`` or ``cli_config`` in place of the platform-specific modules such as ``eos_config``, ``ios_config``, and ``junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
+
+.. note::
+  Network agnostic modules require the ``network_cli`` connection plugin.
+
+
+Sample playbook with platform-specific modules
+----------------------------------------------
+
+This example assumes three platforms, Arista EOS, Cisco IOS, and Juniper JunOS.  Without the network agnostic modules, a sample playbook might contain the following three tasks:
+
+.. code-block:: yaml
+
+  ---
+  - name: RUN ARISTA COMMAND
+    eos_command:
+      commands: show ip int br
+    when: ansible_network_os == 'eos'
+
+  - name: RUN CISCO NXOS COMMAND
```

Comment:
The intro sentence uses Cisco IOS, not NXOS - either that sentence or this task should change for consistency.